recipes/icu/all/conanfile.py

```diff
@@ -114,6 +121,12 @@ def source(self):
     def build(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
+        
+        if self.options.dat_package_file:
+            dat_package_file = glob.glob(os.path.join(self.source_folder, self._source_subfolder, "source", "data", "in", "*.dat"))
+            if dat_package_file:
+                shutil.copy(str(self.options.dat_package_file), dat_package_file[0])
```

Comment:
```suggestion
                selfl.copy(src=str(self.options.dat_package_file), dst=dat_package_file[0])
```

The `self.copy` solves some restrictions on Windows, which `shutil.copy` does not.

Reply:
Does self.copy exist in build(self) ? I think it is not documented

Reply:
In the documentation self.copy(...) is not mentioned for the build(self) method. So I'am not really sure if we should use it.

Reply:
@ivanvurbanov if that file needs to be copied to that location before doing the build, then the copy is fine. Whereas if you are copying the file to that location only for packaging purposes, the copy should be done in the `package()` method